source/numerics/point_value_history.cc

```diff
@@ -734,12 +735,27 @@ ::evaluate_field(const std::vector <std::string> &vector_names, const VECTOR &so
                 }
             }
 
+          //FIXME
+          const Vector< typename VECTOR::value_type >                        &uh_s   = uh[selected_point];
+          const std::vector< Tensor< 1, dim, typename VECTOR::value_type > > &duh_s  = duh[selected_point];
+          const std::vector< Tensor< 2, dim, typename VECTOR::value_type > > &dduh_s = dduh[selected_point];
+          std::vector< Tensor< 1, dim > > tmp_d (duh_s.size());
+          for (unsigned int i = 0; i < duh_s.size(); i++)
+            tmp_d[i] = duh_s[i];
+
+          std::vector< Tensor< 2, dim > > tmp_dd (dduh_s.size());
+          for (unsigned int i = 0; i < dduh_s.size(); i++)
+            tmp_dd[i] = dduh_s[i];
+
+          Vector< double > tmp(uh_s.size());
+          for (unsigned int i = 0; i < uh_s.size(); i++)
+            tmp[i] = uh_s[i];
           // Call compute_derived_quantities_vector
           // or compute_derived_quantities_scalar
           data_postprocessor.
-          compute_derived_quantities_vector(std::vector< Vector< double > > (1, uh[selected_point]),
-                                            std::vector< std::vector< Tensor< 1, dim > > > (1, duh[selected_point]),
-                                            std::vector< std::vector< Tensor< 2, dim > > > (1, dduh[selected_point]),
+          compute_derived_quantities_vector(std::vector< Vector< double > > (1, tmp),
+                                            std::vector< std::vector< Tensor< 1, dim > > > (1, tmp_d),
+                                            std::vector< std::vector< Tensor< 2, dim > > > (1, tmp_dd),
```

Comment:
do we need the tmp vectors above because the data postprocessors are not equipped to deal with anything but doubles? If so, maybe add such a sentence to the FIXME.